### ISSUE TITLE
Object Tracking Hierarchy not expanding with double click

### ISSUE THREAD

**Issue by campmlc** [ArctosDB/arctos#6039]
Issue Documentation is http://handbook.arctosdb.org/how_to/How-to-Use-Issues-in-Arctos.html

**Describe the bug**
Double click to expand the hierarchy tree to show child containers is not working in Find Container.

**To Reproduce**
1. Go to Find Container
2. Enter: DGR in container barcode
3. double click to expand to see child containers
4. See error -nothing happens, just says "Working"

**Expected behavior**
Double click on any level of the hierarchy should open next level of child containers (unless of course it has been opened previously, and then you have to use the tiny + to re-expand)

**Screenshots**

![Screenshot 2023-03-21 16 21 41](https://user-images.githubusercontent.com/14808196/226755136-1fe6f9ee-03d2-41b9-8d06-417f1738043b.png)

**Priority**
High. Impeding work.

**Comment by campmlc** [comment]
Upping priority. I am now not able to find anything by barcodes either in Find Container (by expanding to see the collection object) or by searching by barcode in the main Search menu.
![Screenshot 2023-03-22 15 51 21](https://user-images.githubusercontent.com/14808196/227047045-30f463a0-0852-4eb1-bf34-e8672a5f3dd4.png)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I am having this issue too.

**Comment by campmlc** [comment]
Help - my students and I need this today in order be able to work. Otherwise I'm sending everyone home. Can we get it fixed please?

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Here is a short clip of me trying to load barcode NMMNH_GEO_VERTS.

https://user-images.githubusercontent.com/47402809/227283741-6502aecc-4ff3-4cd2-9a57-607a1a43166d.mp4

It won't load at all, no matter how long I wait.

**Comment by dustymc** [comment]
Are you sharing a network?? I think maybe MSB has blocked this sort of thing before??


I DID NOT do this before taking the screenshots and I'm relatively sure I've never stumbled on this container so nothing would have been cached.

First load is a third of a second.
<img width="1652" alt="Screenshot 2023-03-23 at 10 31 52 AM" src="https://user-images.githubusercontent.com/5720791/227290460-d54b285e-52f7-449c-bf34-4a8c314726f2.png">


expansion is a fourth

<img width="1661" alt="Screenshot 2023-03-23 at 10 33 31 AM" src="https://user-images.githubusercontent.com/5720791/227290578-6292e678-8603-4a08-9601-5bc55bd4aa40.png">

**Comment by KyndallH** [comment]
Working fine for me and has been this entire week.

**Comment by campmlc** [comment]
No, we aren't sharing a network - @Nicole-Ridgwell-NMMNHS and I are in different institutions,locations etc. 
I just see the endless spinning blue dots, and eventually get an error that I terminated the request, but even if I click that OK to make the error box go away, the blue dots keep spinning, and I have to clear form in order to do anything else.

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
I just checked - If I use our wifi instead of the ethernet on my desktop, it works.

**Comment by KyndallH** [comment]
I wonder if clearing your browsing history/cookies would help? @campmlc

**Comment by campmlc** [comment]
Most interesting. Expand hierarchy now seems to be working, after I cleared my cache on my personal computer. However, it is also working now on the DGR computers, where it wasn't working yesterday, and whose cache I did not clear. And neither my computer nor the DGR computers have any connection whatsoever to the NMMNHS computer network.
@dusty did you kick a tire? Or maybe aliens . . . 
However, the main search page Search by Any Barcode is still not working.

**Comment by dustymc** [comment]
I haven't been near any tires, must be ET! (Search by Any Barcode still needs a new issue so it doesn't get lost, that's probably just CPU, a realistic example would be most useful.)

**Comment by Nicole-Ridgwell-NMMNHS** [comment]
Clearing my cache also worked!
